vars/notification.groovy

```diff
@@ -22,3 +22,26 @@ def send_notification_infra(subject, mail_body, version){
     Your RHV-QE ${version} testing pipeline.
     """
 }
+
+def notificator(failed_build, failed_job) {
+  def failed_stage = other.parse_url(failed_build)
+  def build_base_name = other.parse_url(failed_job, true)
+  def stage_link = "<a href=${failed_build}>${failed_stage}</a>"
+  def job_link = "<a href=${failed_job}"+"consoleFull"+">${build_base_name}</a>"
+  def retrigger_link = "<a href=${failed_build}"+"rebuild/parameterized"+">here</a>"
+  def recipient = other.get_team_email(failed_build)
```

Comment:
def team_email = other.get_team_email(failed_build)
def recipient = team_email.contains("rhv-qe-devops")? team_email : "${team_name},rhv-qe-devops"

As we are using rhv-qe-devops on multiple places, I would define it somewhere as variable which we can use on all places. In case we change mail address we can change it on one place ;)